console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2135,8 +2133,34 @@ function updateIotDevice(iotProject, device) {
   return iotdevices[pjIndex].devices[devIndex];
 }
 
+function patchIotDevice(iotProject, id, jsonPatch) {
+  let pjIndex = getIotDevicesIndexForProject(iotProject);
+  let devIndex = iotdevices[pjIndex].devices.findIndex(d => d.deviceId === id);
+
+  if (devIndex < 0) {
+    return {
```

Comment:
You should through a error rather than return object, as we had in other patch queries. Error format should be same as we had. In UI we have a common function for handling the server error, if you will change the error format it will cause issue in UI.